Week2/MohammadDarwesh/mohammaddarwesh.js

```diff
@@ -0,0 +1,180 @@
+// for the first task i will do it with many ways just for practising.
+console.log ("Halo, dunia! ,Ciao, mondo! ,Hola, mundo!");
+
+let indonesian = "Halo, dunia!";
+let italian = "Ciao, mondo!";
+let spanish = "Hola, mundo!";
+console.log(indonesian ,italian, spanish );
+
+let helloWorld = [`${indonesian}` ,`${italian}` ,`${spanish}`];
+console.log(helloWorld);
+console.log(helloWorld[0]);
+console.log(helloWorld[1]);
+console.log(helloWorld[2]);
+
+let helloWorld2 = ["Halo, dunia!" ,"Ciao, mondo!" ,"Hola, mundo!"];
+console.log(helloWorld2);
+console.log(helloWorld2[0]);
+console.log(helloWorld2[1]);
+console.log(helloWorld2[2]);
+console.log(helloWorld2.join("-"));
+console.log(helloWorld2.reverse());
+
+let helloWorld3 = {
+    indonesian: "Halo, dunia!",
+    italian: "Ciao, mondo!",
+    spanish: "Hola, mundo!"
+   };
+console.log(helloWorld3);
+console.log(helloWorld3.indonesian);
+console.log(helloWorld3.italian);
+console.log(helloWorld3.spanish);
+// end of task 1.
+
+
+// task 2:
+console.log("I'm awesome");
+
+// task 3:
+let x;
+console.log("the value of my variable x will be: 32");
+console.log(x);
+x = 32;
+console.log("x will be :32 ");
+console.log(x);
+
+// task 4:
+let y = "Hello, Yash"
+console.log("the value of my string will be: Hello, Yash ");
+console.log(y);
+y = "Hiiii, Yash"
+console.log("the value of my string will be: Hiiii, Yash ");
+console.log(y);
+
+// task 5:
+let z = 7.25;
+console.log(z);
+let a = Math.round(z);
+console.log(a);
+console.log(a > z , z > a );
+if (z > a) {
+    let m = z;
+    console.log(m);
+}
+else{ 
+    m = a;
+    console.log(m);}
+
+
+// task 6:
+let animals;
+console.log("the value of my array will be: cat, dog, hamster");
```

Comment:
Hi Muhammad, your `animals` array was supposed to be created as an empty array. However, like this, `animals` isn't assigned to any value and hence would be `undefined` when you log it.

You were needed to create an empty array i.e `const animals = [];`